frontend/sampledata/luquillo/luq_Species_correctedheaders.txt

```diff
@@ -0,0 +1,76 @@
+spcode,species,family,authority,idlevel,syn,subspecies
```

Comment:
We need to remove all real data from git history.

Reply:
deleted from remote and added to gitignore for future proofing too!